corehq/apps/users/static/users/js/roles.js

```diff
@@ -96,7 +96,6 @@ hqDefine('users/js/roles',[
                         };
                     }),
                 };
-                data.permissions.download_reports = true;  // the user must explicitly disable this when visible
```

Comment:
Is the reverse of this now the case?  Ie, if I grant a role access to a report, will I also have to manually check the "download reports" permission to allow that?  In my opinion, granting access to a report should auto-check this permission too (that's what I was attempting when I initially implemented this).  This is certainly better than the current behavior, though.

Reply:
It won't get auto-checked if you grant access to a report. I did start looking at doing that but it's a bit more complicated since we only want to do that if the user previously didn't have access to any reports which requires maintaining more state.